### DISCUSSION TITLE
yim menu help (⌐■_■)

### DISCUSSION THREAD

**Discussion by yaboisewy** [YimMenu/YimMenu#3424] [General]
so i reset the menu for other reasons and i used to be able to press x i think and it would bring up a a little menu above players head so i could launch them or copy vehicle and other things and now when i press x it doesnt do anything and i cant find how to do it and i was wondering if there is a setting for this ?

**Comment by lonelybud** [comment]
check what key you have set for vehicle duck in game key bindings settings. Use that.

**Reply by yaboisewy** [reply]
yeah just checked and it x still and when i press it it still does nothing and ive tried rebinding and doesnt work still

**Reply by lonelybud** [reply]
select all the options in context menu dialog like vehicle peds objects etc

**Reply by yaboisewy** [reply]
sorry but options where in gta interaction menu or gta settings or menu settings or idk what context menu dialog

**Reply by lonelybud** [reply]
click the blue options button as shown here https://github.com/YimMenu/YimMenu/issues/1713
